src/App.js

```diff
@@ -1,12 +1,37 @@
+import { Routes, Route, Navigate} from 'react-router-dom';
+import { useSelector } from "react-redux";
+
+import CreateAccount from './pages/CreateAccount/CreateAccount';
+import LogInPage from './pages/LogIn/LogInPage';
+
+import RestorePassword from './pages/RestorePassword/RestorePassword';
+import Private from './Komponents/PrivateRoutes/Routes/PrivateRoute';
```

Comment:
remove `Routes` folder and remane `PrivateRoutes` to `PrivateRoute`
chage `Komponents` to `Components` as well